report/sections/lessons-learned-perspective/reflection.tex

```diff
@@ -0,0 +1,21 @@
+\subsection{Reflection}
+We believe we have achieved a high flow rate in our project, by fully automating the process from code to server to ``customer''.
+This has allowed us to work incrementally change our system, and get quick feedback from the customers and monitoring systems.
+Our automation systems also allows for easy rollback of any changes that may have caused issues.\\
+
+This way of deploying often instead of large deployments is very different from our usual projects. As they would usually be building the project and then releasing the entire thing at once.
+Instead we here start with a project that works and incrementally update, fix, and improve from there, while having it deployed and recieving requests.
+It's an interesting and much more reassuring way of working, as it allows us to instantly determine whether or not our changes had a positive, or negative, effect in a production enviroment.
+Which in turn provides us with a constant feedback loop that we can use to improve the system even more.
+This way of working is worth considering in later projects, though it does require some way of creating system load.\\
```

Comment:
To match linebreaks in the rest of the document.
```suggestion
This has allowed us to incrementally change our system, and get quick feedback from the customers and monitoring systems.
Our automation systems also allows for easy rollback of any changes that may have caused issues.

This way of deploying often instead of large deployments is very different from our usual projects. As they would usually be building the project and then releasing the entire thing at once.
Instead we here start with a project that works and incrementally update, fix, and improve from there, while having it deployed and receiving requests.
It's an interesting and much more reassuring way of working, as it allows us to instantly determine whether or not our changes had a positive, or negative, effect in a production environment.
This, in turn provides us with a constant feedback loop that we can use to improve the system even more.
This way of working is worth considering in later projects, though it does require some way of creating system load.
```